LivingSpace/Src/LivingSpace/Classes/UIAlert_LivingSpace.uc

```diff
@@ -61,3 +64,33 @@ simulated protected function CrewOverflowShowLivingRoom ()
 {
 	`HQPRES.CAMLookAtRoom(`XCOMHQ.GetRoom(16), bInstantInterp ? float(0) : `HQINTERPTIME);
 }
+
+simulated protected function string GetCrewOverflowDescription ()
+{
+	local XComGameState_HeadquartersXCom XComHQ;
+	local string Description;
+	local XGParamTag ParamTag;
+
+	ParamTag = XGParamTag(`XEXPANDCONTEXT.FindTag("XGParam"));
+	ParamTag.StrValue0 = string(class'LSHelpers'.default.FACILITY_HOLDS_ENGINEER);
+	ParamTag.StrValue1 = string(class'LSHelpers'.default.FACILITY_HOLDS_SCIENTIST);
```

Comment:
Please fetch the facility template and get the localized name from there. I'm pretty sure the template name and the UI name match only in English (and not even in English when switched to other facilities, e.g. GTS)